handler/src/main/java/io/netty/handler/ssl/Ciphers.java

```diff
@@ -0,0 +1,758 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import io.netty.util.internal.ObjectUtil;
+
+import java.util.Arrays;
+
+/**
+ * Cipher suites
+ */
+public final class Ciphers {
```

Comment:
Are there any of these that should be discouraged?

Reply:
Cipher comes below SSL/TLS protocols. If we deprecate them then ciphers will fall under the same category. So marking ciphers `@Deprecated` is not needed.

Reply:
Okay, sounds good.